en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -88,34 +87,138 @@ class MyDataObject extends DataObject
+[/hint]
+
+**Globally change the primary search field name via yaml config**
+```yml
+SilverStripe\ORM\DataObject:
+  primary_search_field_name: 'my_primary_field_name'
+```
+
+**Customise the primary search field name via yaml _or_ php config**
+```php
+use SilverStripe\ORM\DataObject;
+
+class MyDataObject extends DataObject
+{
+    private static string $primary_search_field_name = 'my_primary_field_name';
+}
+```
+
+#### Specify a search filter for primary search
+
+By default, the primary search will search across your fields using a [PartialMatchFilter](api:SilverStripe\ORM\Filters\PartialMatchFilter) regardless of what filters you have [specified for those fields](#specify-a-form-field-or-search-filter).
+
+You can configure this to be a specific filter class, or else disable the primary search filter. Disabling the filter will result in the filters you have specified for each field being used when searching against that field in the primary search.
+
+Like the primary search field name, you can set this either globally or per class:
+
+**Globally change the primary search filter via yaml config**
+```yml
+# use a specific filter
+SilverStripe\ORM\DataObject:
+  primary_search_field_filter: 'SilverStripe\ORM\Filters\EndsWithFilter'
+
+# or disable the filter to fall back on individual fields' filters
+SilverStripe\ORM\DataObject:
+  primary_search_field_filter: ''
+```
+
+**Customise the primary search filter via yaml _or_ php config**
+```php
+use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\Filters\EndsWithFilter;
+
+class MyDataObject extends DataObject
+{
+    private static string $primary_search_field_name = EndsWithFilter::class;
+}
+```
+
+[warning]
+You may get unexpected results using some filters if you don't disable splitting the query into terms - for example if you use an [ExactMatchFilter](api:SilverStripe\ORM\Filters\ExactMatchFilter), each term in the query _must_ exactly match the value in at least one field to get a match. If you disable splitting terms, the whole query must exactly match a field value instead.
+[/warning]
+
+#### Splitting search queries into individual terms
+
+By default the primary search field qill split your search query into individual terms, and search across your searchable field for each term. At least one field must match each term to get a match.
+
+For example: with the search query "farm house" at least one field must have a match for the word "farm", and at least one field must have a match for the word "house". There does _not_ need to be a field which matches the full phrase "farm house".
+
+You can disable this behaviour by setting `DataObject.primary_search_split_terms` to false. This would mean that for the example above a `DataObject` would need a field that matches "farm house" to be included in the results. Simply amtching "farm" or "house" alone would not be sufficient.
```

Comment:
```suggestion
You can disable this behaviour by setting `DataObject.primary_search_split_terms` to false. This would mean that for the example above a `DataObject` would need a field that matches "farm house" to be included in the results. Simply matching "farm" or "house" alone would not be sufficient.
```